src/provable/field-bigint.ts

```diff
@@ -122,3 +134,37 @@ function ProvableBigint<
     },
   };
 }
+
+function BinableBigint<T extends bigint = bigint>(
+  sizeInBytes: number
+): Binable<T> {
+  return {
+    toBytes(x) {
+      return bigIntToBytes(x, sizeInBytes);
+    },
+    fromBytes(bytes) {
+      return bytesToBigInt(bytes) as T;
+    },
+    sizeInBytes() {
+      return sizeInBytes;
+    },
+  };
+}
+
+function bytesToBigInt(bytes: Uint8Array | number[]) {
+  let x = 0n;
+  let bitPosition = 0n;
+  for (let byte of bytes) {
+    x += BigInt(byte) << bitPosition;
+    bitPosition += 8n;
+  }
+  return x;
+}
+
+function bigIntToBytes(x: bigint, length: number) {
```

Comment:
do you have unit tests on these somewhere?

Reply:
can you add them if mising? Ideally we'd have some quickcheck-type library and test `forall b : Bigint, exists length: number, such that bytesToBigInt(bigIntToBytes(b, length)) == b`

Reply:
in lieu of that, just pick a few examples -- negative numbers, 0, numbers that fit in one limb, bigger numbers